proto/eventlog.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+import "proto/context.proto";
+
+package eventlog;
+
+option go_package = "proto;eventlog";
+
+message GetEventLogChunkRequest {
+  // The request context.
+  context.RequestContext request_context = 1;
+
+  // The invocation identifier itself.
+  string invocation_id = 2;
+
+  message OptionalChunklID {
+    int32 value = 1;
+  }
+
+  // Optionally specify the chunk to be returned; gets the last chunk if unset
+  OptionalChunkID chunk_id = 3;
+}
+
+message GetEventLogChunkResponse {
+  // The response context.
+  context.ResponseContext response_context = 1;
+
+  message Chunk {
+    int32 chunk_id = 1;
+    bytes buffer = 2;
+  }
+
+  // The requested chunk
+  Chunk chunk = 2;
+}
```

Comment:
if you add pointers to prev and next you'll have the freedom to change the implementation without changing the clients. Also do things like collapse consecutive chunks without telling the client.